frontend/src/components/Chat/ChatContainer.js

```diff
@@ -110,6 +121,7 @@ const ChatContainer = ({ chatsData, participants, currentUser, onClick }) => {
               height: "80%",
               overflowY: "auto",
             }}
+            ref={(el)=> dialogContainer = el }
```

Comment:
in theory you can use useRef react hook (there are some places in the project). Then you can use just ```ref={dialogContaner}```

Reply:
there is one place - DoctorAvailabilityCalendar, but it works :)